src/macaron/slsa_analyzer/specs/ci_spec.py

```diff
@@ -38,5 +38,5 @@ class CIInfo(TypedDict):
     Schema: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release.
     """
 
-    provenances: Sequence[InTotoPayload]
-    """The JSON payloads of SLSA provenances in in-toto format."""
+    provenances: Sequence[SLSAProvenanceData]
```

Comment:
Shouldn't we use the `DownloadedProvenanceData` interface instead of the specific class `SLSAProvenanceData` here?
```suggestion
    provenances: Sequence[DownloadedProvenanceData]
```
 Provenances may not all be SLSA provenances. In addition, the `DownloadedProvenanceData` interface is also being used in the `PackageRegistryInfo` class which is a parallel class to `CIInfo`.

Reply:
See commit [d25bf57](https://github.com/oracle/macaron/pull/620/commits/d25bf57f3e6f57d29cf4c311207dbcfe1302727e).